config/default.yml

```diff
@@ -248,6 +248,8 @@ Performance/StartWith:
 Performance/StringInclude:
   Description: 'Use `String#include?` instead of a regex match with literal-only pattern.'
   Enabled: 'pending'
+  AutoCorrect: false
```

Comment:
isn't it okay to leave `AutoCorrect: true`, as long as `SafeAutoCorrect: false`?

my understanding is:

- `AutoCorrect: true` and `SafeAutoCorrect: true`= autocorrected by `rubocop -a` or `rubocop -A`
- `AutoCorrect: true` and `SafeAutoCorrect: false`= autocorrected by `rubocop -A` only
- `AutoCorrect: false` and `SafeAutoCorrect: false`= never autocorrected
- `AutoCorrect: false` and `SafeAutoCorrect: true`= never autocorrected

Currently this defaults to the third option. Why not the second?

Reply:
As it stands, `AutoCorrect: false` in the default file is reserved for Cops that have corrections that could be useful to some but probably not most people. An example that comes to my mind is `LineLength`.

This setting should be overriden in `.rubocop.yml` or via CLI for those that want it